apps/data_handler/db/models/zklend_events.py

```diff
@@ -39,3 +39,80 @@ class LiquidationEventData(EventBaseModel):
     debt_face_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
     collateral_token: Mapped[str] = mapped_column(String, nullable=False)
     collateral_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
+
+
+class RepaymentEventData(EventBaseModel):
+    """
+    Database model for Repayment event, inheriting from EventBaseModel.
+
+    This model stores details of a repayment event, including the addresses involved
+    and the amounts in raw and face formats.
+    """
+
+    __tablename__ = "repayment_event_data"
+
+    repayer: Mapped[str] = mapped_column(String, nullable=False)
+    beneficiary: Mapped[str] = mapped_column(String, nullable=False)
+    token: Mapped[str] = mapped_column(String, nullable=False)
+    raw_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
+    face_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
+
+
+class DepositEventData(EventBaseModel):
+    """
+    Database model for Deposit event, inheriting from EventBaseModel.
+
+    This model stores details of a deposit event, including the user address,
+    token, and the face amount of the deposit.
+    """
+
+    __tablename__ = "deposit_event_data"
```

Comment:
we don't need to have here `_data`. please remove it
`deposit_event` will be okay